scripts/plot_dataframe.py

```diff
@@ -83,30 +282,35 @@ def plot_analyzed_dataframe(args: Namespace) -> None:
     )
 
     df_buy = data[data['buy'] == 1]
+
     buys = go.Scattergl(
         x=df_buy.date,
-        y=df_buy.close,
+        y=df_buy.close * 0.995,
```

Comment:
I do understand that moving this point (and the sell-point too) slightly would increase readability - but won't that then show "wrong" in the mouseover-hint?